### DISCUSSION TITLE
希望增加用特定编程语言实现了的算法描述

### DISCUSSION THREAD

**Discussion by FireJohnny** [doocs/leetcode#267] [Ideas] [answered]
表示leetcode的第一题，java实现完成，python实现完成，C++待补充等

**Comment by yanglbme** [comment]
会增加的，目前相对完善的是剑指 Offer 系列题目，比如：https://github.com/doocs/leetcode/tree/master/lcof/%E9%9D%A2%E8%AF%95%E9%A2%9803.%20%E6%95%B0%E7%BB%84%E4%B8%AD%E9%87%8D%E5%A4%8D%E7%9A%84%E6%95%B0%E5%AD%97

其他题解会陆续更新上去，也欢迎朋友们来贡献题解。

**Comment by yanglbme** [comment, marked as answer]
后续可以更新为类似下面这种：

### 题解
- [x] Java
- [x] Python
- [ ] C++

**Reply by Dohnbl** [reply]
- - [ ] Nice_[work]()_****
